src/coreclr/nativeaot/Runtime/amd64/GcProbe.asm

```diff
@@ -140,7 +140,6 @@ LEAF_END RhpGcPoll, _TEXT
 
 NESTED_ENTRY RhpGcPollRare, _TEXT
         PUSH_COOP_PINVOKE_FRAME rcx
-        END_PROLOGUE
```

Comment:
Fixing unrelated build warning (END_PROLOGUE is called by PUSH_COOP_PINVOKE_FRAME, so this one is redundant)

Reply:
Those should obviously be fixed too even if it makes no codegen difference (`END_PROLOGUE` is no-op on non-Windows).